pkg/virt-controller/watch/migration/migration.go

```diff
@@ -979,6 +981,13 @@ func (c *Controller) handleTargetPodHandoff(migration *virtv1.VirtualMachineInst
 	if !c.isMigrationPolicyMatched(vmiCopy) {
 		vmiCopy.Status.MigrationState.MigrationConfiguration = clusterMigrationConfigs
 	}
+	// TODO: disable post-copy for volume migration untill is properly tested
+	if volmig.IsVolumeMigrating(vmiCopy) &&
+		vmiCopy.Status.MigrationState.MigrationConfiguration != nil &&
+		*vmiCopy.Status.MigrationState.MigrationConfiguration.AllowPostCopy {
```

Comment:
I would not even check the actual value, just set it to false, logically it is the same.

Reply:
Done